formidable/app.py

```diff
@@ -21,3 +23,11 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
+
+        # Deprecation warning for django 1.8 and 1.9
+        if django.VERSION[0] is 1:
```

Comment:
you'd better not use this... Please use version comparison tools, like: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python